src/ios/registerNativeModule.js

```diff
@@ -78,6 +149,14 @@ module.exports = function registerNativeModuleIOS(dependencyConfig, projectConfi
     });
   });
 
+  const headers = getHeadersInFolder(dependencyConfig.folder);
+  if (headers.length > 0) {
```

Comment:
No action is taken when library exposes purely JS solution (e.g. react-native-side-menu)